ztp/siteconfig-generator/siteConfig/clusterCRsV1.go

```diff
@@ -0,0 +1,185 @@
+package siteConfig
+
+// SiteConfigV1 and ClusterCRsV1 are deprecated use SiteConfigV2 and ClusterCRsV2 to add new field.
+// SiteConfigV1 uses clusterCRsV1 to generate the required CRs for cluster provisioning
+
+const siteConfigAPIV1 = "ran.openshift.io/v1"
+
+const clusterCRsV1 = `
```

Comment:
Is this v1/v2 split a good time to consider using [go:embed](https://pkg.go.dev/embed) instead of hard-coding strings in code?

Reply:
not sure what id the benfites of using go:embed rather than defining the version as const.  will separate the version from the apiGroup.

Reply:
The benefit is that the actual template CRs would be in separate files in the filesystem instead of just inline inside a go module.